src/CIQS-CLI-Extension/azext_ciqs/custom.py

```diff
@@ -12,16 +12,40 @@
 from azure.cli.core._profile import Profile
 from azure.cli.core.util import in_cloud_console
 
+import json
+import http.client
+
 logger = get_logger(__name__)
 
+#---------------------------------------------------------------------------------------------
 # group ciqs
-def locations(cmd):
-    print('To be implemented')
+#---------------------------------------------------------------------------------------------
+
+HOST = 'ciqs-api-westus.azurewebsites.net'
 
-# group ciqs deployments
+def _makeAPICall(cmd, method, path, auth_token=None):
```

Comment:
Put helper methods in another file, or class. Group them accordingly if necessarily.